lib/ansible/utils/module_docs_fragments/eos.py

```diff
@@ -120,6 +120,11 @@ class ModuleDocFragment(object):
             on personally controlled sites using self-signed certificates.  If the transport
             argument is not eapi, this value is ignored.
         choices: ['yes', 'no']
+      use_proxy:
+        description:
+          - If C(no), proxy environment variables (http_proxy/https_proxy) will be ignored.
```

Comment:
`If C(no), the environment variables C(http_proxy) and C(https_proxy) will be ignored.